### PR TITLE
Merging 2.0.0 changes from dotnet-docker-nightly

### PR DESCRIPTION
**Pull request by MichaelSimons**
Note: It is expected that 2.0 CI will fail until the 2.0 NuGet packages are available on the public feeds.  This PR won't be merged until this happens.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -17,258 +17,410 @@
...
-          "sharedTags": [
-            "1.0.5-sdk-1.0.4",
-            "1.0.5-sdk",
-            "1.0-sdk"
-          ],
-          "platforms": {
-            "linux": {
+          "readmeOrder": 2,
+          "sharedTags": {
+            "1.0.5-sdk-1.1.0": {
```
Comment (dagood) [inline manifest.json]: I see a change from SDK 1.0.4 to 1.1.0 in the 1.0/1.1 tags, but the Dockerfiles don't appear to have the version change. Was https://github.com/dotnet/dotnet-docker-nightly/commit/c32bbf0980859f51b958c000209127c3e979569f partially applied?

File: build-pipeline/dotnet-docker-linux-arm32v7-images.json [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+      "condition": "succeeded()",
+      "refName": "Task3",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "run $(docker.commonRunArgs) $(docker.gitEnabledImageName) git clone https://$(PB.PAT.dn-bot-devdiv-build-rw-code-rw)@github.com/dotnet/dotnet-docker.git /repo",
```
Comment (crummel) [inline build-pipeline/dotnet-docker-linux-arm32v7-images.json]: Are we planning on making this repo private at some point?  This seems like one more thing to maintain if it's never going to be needed.

File: manifest.json (hunk 1 above)
Comment (MichaelSimons) [inline manifest.json]: The manifest change is incorrect.  Reverting the 1.0.4 to 1.1.0 change.

File: build-pipeline/dotnet-docker-linux-arm32v7-images.json (hunk 2 above)
Comment (dagood) [inline build-pipeline/dotnet-docker-linux-arm32v7-images.json]: Hmm, this does seem odd. `dn-bot-devdiv-build-rw-code-rw` is for our VSTS repos (if the name is right), and I'm not sure what the behavior is when using it on GitHub.

Comment (MichaelSimons) [inline build-pipeline/dotnet-docker-linux-arm32v7-images.json]: Wonky indeed and isn't being specified by PB in my test builds.  Removed.

**Comment by MichaelSimons** [comment]
@dotnet-bot test Debian - 2.0 Dockerfiles please

**Pull request merged**